model/lesson.js

```diff
@@ -122,6 +139,40 @@ async function getLessonById(data) {
     );
 }
 
+/*Fetches verses in a lesson based*/
+//ASSUMPTIONS: Verses in a Lesson are contigous
+async function getLessonVerses(data) {
+    console.log(data);
+    var invalid = utils.simpleValidation(data, {
+        lesson_id: "integer",
+    });
+    if (invalid) {
+        return invalid;
+    }
+
+    let lesson = await getLessonById(data);
```

Comment:
good stuff, I like the use of helper functions. makes it very func-y... ha. get it?

Reply:
nope, but still laughed